convert2rhel/checks.py

```diff
@@ -284,13 +284,20 @@ def ensure_compatibility_of_kmods():
                 unsupported_kmods,
             )
         )
-        logger.critical(
-            (
+        if "CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS" in os.environ:
+            logger.warning(
+                "The following kernel modules are not supported in RHEL:\n{kmods}\n"
+                "'CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS' environment variable detected, continuing conversion."
+            ).format(kmods=not_supported_kmods, system=system_info.name)
```

Comment:
Couple of suggestions here… 

First, the `system` parameter in the `format()` is not being used in the string message, did you forget to add this variable in the `str`, or it is not necessary? I would argue that this is not needed at all, and you can remove it. 

```suggestion
            logger.warning(
                "The following kernel modules are not supported in RHEL:\n{kmods}\n"
                "'CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS' environment variable detected, continuing conversion."
            ).format(kmods=not_supported_kmods)
```

Second, I would rephrase this message to output the detection of the environment variable first and then output the kernel modules, like this: 

```suggestion
            logger.warning(
                "Detected 'CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS' environment variable, we will continue the conversion even with the following kernel modules that are not supported: ".
                "Kernel modules that are not supported in RHEL:\n{kmods}\n"
            ).format(kmods=not_supported_kmods, system=system_info.name)
```

Feel free to re-arrange this message if you (or any other reviewer) think that it could benefit from a diferent message.

Reply:
Thank you for this great suggestion!, I will definitely re-write this changed!